python/ray/tests/test_actor_failures.py

```diff
@@ -891,6 +892,141 @@ def ping(self):
     assert wait_for_condition(lambda: ray.get(parent_actor.wait.remote()))
 
 
+@pytest.mark.parametrize(
+    "ray_start_cluster", [{
+        "num_cpus": 1,
+        "num_nodes": 1,
+    }], indirect=True)
+def test_location_resolution_worker_failure(ray_start_cluster):
```

Comment:
We should update the logic and wording of this test. Because we don't have `location resoloution protocol` now.

Reply:
Should just use something like `test_actor_owner_dies_before_dependency_ready`.